doc/admin_doc/install_doc/installation.rst

```diff
@@ -186,13 +186,18 @@ To keep a clean distribution, we recommend to work with a virtualenv:
     python3 -m venv /path/to/venv/INGInious
     source /path/to/venv/INGInious/bin/activate
 
-The recommended setup is to install INGInious via pip.
-This allows you to use the latest released version. This version is currently the supported one for issues.
+Then install INGInious using the setup.py file
 
-::
+.. code-block:: bash
 
-    $ pip install INGInious
+    pip install setuptools
```

Comment:
This shouldn't be necessary. For deployments `pip install inginious` remains the way to go.
Developers can do `pip install .`